### ISSUE TITLE
Global option and option for specific feeds, to automatically mark items older than x days as read 

### ISSUE THREAD

**Issue by ghost** [FreshRSS/FreshRSS#824]
**Feature request**
On a global level it would be useful to automatically mark feed items that are older than x days (e.g. 7 days). This should not be a default setting of course. 

News feeds with a high volume are quickly out-of-date. This in contrast to feeds with e.g. background stories, long reads and (personal) blogs. I like to mark news feeds automatically as read after e.g. one or two days. A way to do that is to provide an option for specific feeds to automatically mark items older than x days.

**Comment by Alkarex** [comment]
Yes. So far, it is only possible to do that manually from the user interface, with the drop-down menu "mark as read, older than 1 week" or "older than 1 day".

**Comment by ghost** [comment]
Yes I know, that's why I created this feature request.
